infra/scenarios/bicep-private-registry/client.bicep

```diff
@@ -0,0 +1,8 @@
+// Azure resources using Bicep modules from the private registry
+// FIXME: override the Azure Container Registry name
```

Comment:
use module alias ref. [Add module settings in the Bicep config file](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-config-modules#aliases-for-modules)